app/src/features/Batches/BatchCreate.tsx

```diff
@@ -0,0 +1,184 @@
+    width: 250,
+  },
+}));
+
+const BatchCreate = (): JSX.Element => {
+  const { t } = useTranslation();
+
+  const classes = useStyles();
+
+  const [selectedResourceIds, setSelectedResourceIds] = useState<string[]>([]);
+
+  const [alert, setAlert] = useState<string | undefined>(undefined);
+  const handleAlertClose = () => setAlert(undefined);
+
+  const { sourceId: id } = useParams<{ sourceId: string }>();
+
+  const { data: resources } = useApiResourcesListQuery(
+    { source: id },
+    { skip: !Boolean(id) }
+  );
+
+  const [apiBatchCreate] = useApiBatchesCreateMutation();
+
+  const handleResourceSelectionChange = (
+    event: React.ChangeEvent<{
+      name?: string;
+      value: unknown;
+    }>
+  ) => {
+    setSelectedResourceIds(event.target.value as string[]);
+  };
+
+  const handleBatchRun = async () => {
+    try {
+      await apiBatchCreate({
+        batchRequest: {
+          resources: selectedResourceIds,
+        },
+      }).unwrap();
+
+      setSelectedResourceIds([]);
+    } catch (e) {
+      setAlert(e.message as string);
+    }
+  };
+
+  return (
+    <div className={classes.root}>
+      <FormControl className={classes.formControl}>
+        <InputLabel id="demo-mutiple-chip-label">{t("resources")}</InputLabel>
+        <Select
+          variant="filled"
+          labelId="demo-mutiple-chip-label"
+          id="demo-mutiple-chip"
+          multiple
+          value={selectedResourceIds}
+          onChange={handleResourceSelectionChange}
+          input={<Input id="select-multiple-chip" />}
+          renderValue={(selected) => (
+            <div className={classes.chips}>
+              {(selected as string[]).map((resourceId) => {
+                const resource = resources?.find(({ id }) => resourceId === id);
+                return (
+                  resource && (
+                    <Chip
+                      size="small"
+                      key={`resource-selected-${resource.id}`}
+                      label={`${resource.definition_id} - ${resource.label}`}
+                      className={classes.chip}
+                    />
+                  )
+                );
+              })}
+            </div>
+          )}
+          MenuProps={{
+            PaperProps: {
+              className: classes.menuPaper,
+            },
+            anchorOrigin: {
+              vertical: "top",
+              horizontal: "left",
+            },
+            transformOrigin: {
+              vertical: "top",
+              horizontal: "left",
+            },
+            getContentAnchorEl: null,
+          }}
+        >
+          {resources &&
+            resources.map(({ id, definition_id, label }) => (
+              <MenuItem
+                key={`resource-option-${id}`}
+                value={id}
+                className={clsx(classes.mediumBold, {
+                  [classes.regularBold]: selectedResourceIds.includes(id),
+                })}
```

Comment:
I have not been able to test it, but I think applying the style this way would be cleaner : 

```suggestion
                classes={{
                  root: classes.mediumBold,
                  selected: classes.regularBold,
                }}
```